netbox/dcim/tests/test_models.py

```diff
@@ -592,6 +597,155 @@ def test_connection_via_single_rear_port(self):
         self.assertIsNone(endpoint_a.connection_status)
         self.assertIsNone(endpoint_b.connection_status)
 
+        # Recreate cable 1 to test creating the cables in reverse order (RP first, FP second)
+        cable1.save()
```

Comment:
It may be preferable to recreate the cable explicitly in the interest of clarity. Although it technically works, calling `save()` on an object after calling `delete()` may be confusing to the reader.